clang/test/lab1/akopyan_zal/Test.cpp

```diff
@@ -0,0 +1,11 @@
+// RUN: %clang_cc1 -ast-dump -ast-dump-filter -load %llvmshlibdir/AlwaysInlinePluginAkopyan%pluginext -plugin always-inline %s 2>&1 | FileCheck %s

```

Comment:
Here name should be changed too:
```suggestion
// RUN: %clang_cc1 -ast-dump -ast-dump-filter -load %llvmshlibdir/AlwaysInlinePluginAkopyan%pluginext -plugin always_inline %s 2>&1 | FileCheck %s
```

Reply:
done